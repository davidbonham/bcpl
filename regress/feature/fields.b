GET "libhdr"

MANIFEST $(
    sign = SLCT 1:63:0
$)

LET start() BE $(
    LET b = VEC 0
    LET c = ?
    b!0 := -1
    c := sign OF b
    writen(c)
$)
