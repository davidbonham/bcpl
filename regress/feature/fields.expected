-- stdout -------------------------------------------
0000000000000010
00000000F4F5F6F7
0000000000000001
0070605040302010
-- stderr -------------------------------------------

